### PR TITLE
Solve FindFreetype dependencies

### PR DESCRIPTION
**Pull request by uilianries**
Related issue: https://github.com/bincrafters/community/issues/216

When FindFreetype is consumed, it dependencies, as libpng, are not listed on FREETYPE_LIBRARY, so any project MUST use CONAN_LIBS to solve all libraries.

However, it's possible to manipulate FindFreetype to get all dependencies and fill FREETYPE_LIBRARY as well.

/cc @memsharded @lucienboillod

### CODE REVIEW

**Review by memsharded — APPROVED** [review]
Looks good!

File: FindFreetype.cmake [hunk 1]
```diff
@@ -1,19 +1,21 @@
+cmake_minimum_required(VERSION 2.8)
 
+foreach(libname ${CONAN_LIBS})
+    find_library(${libname}_LIBRARY NAMES ${libname} PATHS ${CONAN_LIB_DIRS} NO_CMAKE_FIND_ROOT_PATH)
```
Comment (lucienboillod) [inline FindFreetype.cmake]: It should collect only freetype dependencies in FREETYPE_LIBRARY.

**Review by lucienboillod — CHANGES_REQUESTED** [review]

File: FindFreetype.cmake (hunk 1 above)
Comment (uilianries) [reply FindFreetype.cmake]: conanbuildinfo.cmake only list freetype and libm as dependencies, so libpng, zlib and bzip2 are separated. I didn't find a smart way to solve it.

**Comment by uilianries** [comment]
@memsharded @lucienboillod I updated the branch, please check again.

As you can see, I used find_library to solve not just freetype, but also libpng, zlib, bzip2 and libm. I didn't find a smart way to solve freetype dependencies.

I could set FREETYPE_LIBRARY=${CONAN_LIBS}, the result will be `freetype;libpng;zipb;m;bzip2`, however we have no control about witch files cmake will chose.

**Review by memsharded — APPROVED** [review]
Yes, this looks better and more robust now.

**Comment by uilianries** [comment]
Thanks for review!

**Pull request merged**